### ISSUE TITLE
Sending notifications

### ISSUE THREAD

**Issue by accarniel** [QCB-Collaboratory/telescope#5]
Is it possible to send notifications (e.g., an email) if one machine is down? That is, when a machine was rebooted or turned off.

**Comment by thmosqueiro** [comment]
Yes! We were planning on implementing notifications when jobs finish. But a notification when a machine is down is a great idea.

This can probably be done using the ssh status. We can probably detect unreachable servers and rejected connections, which may be symptoms of different issues. @thotypous Any other ideas?
